src/org/scada_lts/web/mvc/api/SystemSettingsAPI.java

```diff
@@ -56,6 +56,46 @@
         }
     }
 
+    @GetMapping(value = "/getDefaultLoggingType", produces = "application/json")
+    public ResponseEntity<String> getLoggingType(HttpServletRequest request) {
+        LOG.info("/api/systemSettings/getDefaultLoggingType");
+        try {
+            User user = Common.getUser(request);
+            if (user != null && user.isAdmin()) {
+                ObjectMapper mapper = new ObjectMapper();
+                try {
+                    String json = mapper.writeValueAsString(systemSettingsService.getDefaultLoggingType());
+                    return new ResponseEntity<>(json, HttpStatus.OK);
+                } catch (JsonProcessingException e) {
+                    LOG.error(e);
+                    return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+                }
+            } else {
+                return new ResponseEntity<>(HttpStatus.UNAUTHORIZED);
+            }
+        } catch (Exception e) {
+            LOG.error(e);
+            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+        }
+    }
+
+    @PostMapping(value = "/saveDefaultLoggingType/{defaultLoggingType}")
+    public ResponseEntity<String> saveLoggingType(HttpServletRequest request, @PathVariable("defaultLoggingType") String defaultLoggingType) {
+        LOG.info("/api/systemSettings/saveLoggingType");
+        try {
+            User user = Common.getUser(request);
+            if (user != null && user.isAdmin()) {
+                systemSettingsService.setDefaultLoggingType(defaultLoggingType);
+                return new ResponseEntity<>(SAVED_MSG, HttpStatus.OK);
+            } else {
+                return new ResponseEntity<>(HttpStatus.UNAUTHORIZED);
+            }
+        } catch (Exception e) {
+            LOG.error(e);
+            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
```

Comment:
Bad request relates to invalid data sent to api. May be returned after validation of input arguments, for other errors it is probably better to return 500.